internal/cli/atlas_alerts_config_update.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"github.com/mongodb/mongocli/internal/flags"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type atlasAlertsConfigUpdateOpts struct {
+	*atlasAlertsConfigCreateOpts
```

Comment:
[nit] I'm not sure if I like this, we now have `atlasAlertsConfigUpdateOpts` implementing two `init`s and two `Run` methods, If the point is to reuse `buildAlertConfiguration` let's create a third `*Opts` struct and embed that in both 
`atlasAlertsConfigUpdateOpts` and `atlasAlertsConfigUpdateOpts`

Reply:
you can place that struct in the cli.go file which is kind of our dumping ground for these helpers